index-import.cpp

```diff
@@ -173,9 +173,16 @@ static IndexUnitWriter remapUnit(const std::unique_ptr<IndexUnitReader> &reader,
   });
 
   reader->foreachInclude([&](const IndexUnitReader::IncludeInfo &info) {
-    // Note this isn't revelevnt to Swift.
-    writer.addInclude(fileMgr.getFile(info.SourcePath), info.SourceLine,
-                      fileMgr.getFile(info.TargetPath));
+    const auto sourcePath = remapper.remap(info.SourcePath);
+    const auto targetPath = remapper.remap(info.TargetPath);
+    if (Verbose) {
+      *outs << "Include SourcePath: " << sourcePath << "\n"
```

Comment:
```suggestion
      *outs << "IncludeSourcePath: " << sourcePath << "\n"
```

Reply:
The context is that I tried to keep these logs as valid(-ish) yaml. It's not consumed as yaml, but I'd prefer to keep them consistent.

Reply:
Perhaps that should be documented in the source code? A comment maybe?

Reply:
I'm thinking of pulling out the logging and instead adding a `dump-unit` command (name tbd) and using that for tests and using that to introspect before and after. In which case this all goes away.